lab2/src/task1/Complex.java

```diff
@@ -0,0 +1,46 @@
+package task1;
+
+public class Complex {
+    private int R, I;
+
+    public Complex(int _R, int _I) {
```

Comment:
Don't use identifiers starting with _ (it's considered a bad coding style). If you want to find out more, see https://www.ibm.com/docs/en/zos/2.3.0?topic=tokens-identifiers

Just do this. R = R . The "this" word has the role of eliminating confusion between class attributes and parameters with the same name